### PR TITLE
Submit can show problem name

### PR DESCRIPTION
**Pull request by henryyang42**
Submit can show problem name as is suggested by @totoroXD

### CODE REVIEW

File: status/templates/status/statusTable.html [hunk 1]
```diff
@@ -30,8 +30,7 @@
           "success" {% else %} "danger" {% endif %}>
           <td>
             {% if submission.grouper|show_detail:user %}
-              <a target="_blank"
```
Comment (henryyang42) [inline status/templates/status/statusTable.html]: I want to open a discuss that, if we should open a new tab to view code?
If we resubmit again and again, chances are there are many new tabs on the browser , which is pretty annoying.
I don't know if disabling open new tab would be better.

Comment (nickelcheng) [inline status/templates/status/statusTable.html]: We can disable it. If an user requires new tab, they can "open it in a new tab".

**Comment by nickelcheng** [comment]
test ok, +1

**Comment by nickelcheng** [comment]
going to merge.

**Pull request merged**